### DISCUSSION TITLE
Horizontal bar for viewing horizontally long svgs (instead of scaling)

### DISCUSSION THREAD

**Discussion by VarLad** [fonsp/Pluto.jl#3085] [Questions about using Pluto] [answered]
For some SVGs, horizontal scrolling makes more sense than scaling, due to the amount of information displayed in the SVG.
Is there an option (hopefully an HTML script?) which I can use to enable horizontal scrollbar for viewing an svg?

**Comment by fonsp** [comment]
Maybe you can wrap it in a `PlutoUI.ExperimentalLayout.Div([image])`?

**Reply by VarLad** [reply]
Doesn't seem to work, here's what I see:
![image](https://github.com/user-attachments/assets/a13d9897-b760-4b77-b212-5112daebc6fe)

Here's the original image
![image](https://github.com/user-attachments/assets/2407a588-f482-48ff-8725-44a106ac4870)

**Reply by VarLad** [reply]
@fonsp From what I remember, the horizontal scroll used to be the default behavior. Is there any way to revert back to that, and more importantly, something package authors can use use to decide whether they want horizontal scroll in Pluto or not?

**Comment by fonsp** [comment, marked as answer]
# Full width for `PlutoUI.Resource`
If you use `PlutoUI.Resource` or `PlutoUI.LocalResource`, you can override the max-width attribute from Pluto:

```julia
img = PlutoUI.Resource(url, :style => "max-width: unset")
```

# Full width for any image
A more general solution that also works for other types of images:

```julia
full_width(content) = PlutoUI.ExperimentalLayout.Div([img]; style="width: max-content")

# then in another cell:
full_width(my_image_or_plot)
```
